File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
import java.util.Arrays;
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] num = new int[nums1.length+nums2.length];
        for(int i=0; i<nums1.length; i++){
            num[i] = nums1[i];
        }
        for(int i=0; i<nums2.length; i++){
            num[nums1.length+i] = nums2[i];
        }
        Arrays.sort(num);
        int sum = 0;
        double median = 0.0;
        int n = num.length;
        if(n==1) return num[0];
        if(n%2!=0){
            return num[n/2];
        }else{
            median = (double)(num[(n-1)/2]+num[n/2])/2;
            return median;
        }
        
    }
}

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        result = nums1+nums2
        result = sorted(result)
        n = len(result)
        if n%2==0:
            x = result[int(n/2)-1]+result[int(n/2)]
            return x/2
        else:
            return result[int(n/2)]
        return 0.0
        